elasticsearch-jmeter/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,793 @@
+﻿{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+
+    },
+    "variables": {

```

Comment:
many of these variables are only used once - let's just use literal values in those cases... the variables just make the template harder to read.